src/Exception/MissingSettingsException.php

```diff
@@ -2,4 +2,5 @@
 
 namespace Drupal\acquia_lift\Exception;
 
-class MissingSettingsException extends \Exception {}
+class MissingSettingsException extends \Exception {
+}
```

Comment:
Should either 1) be renamed "AcquiaLiftMissingSettingsException", or 2) Inherit from your new "AcquiaLiftException" now.